testing/u-boot-s5p6818/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor:
+# Maintainer:
+pkgname=u-boot-s5p6818
+pkgver=1.2
+pkgrel=0
+pkgdesc="u-boot for s5p6818"
+url="https://github.com/rafaello7/u-boot-nanopi-m3"
+arch="aarch64"
+license="BSD-2-Clause"
+depends=""
```

Comment:
empty variables can be removed for brevity

Reply:
Removed. Thank you!